### PR TITLE
Packages update for WordPress 6.7 RC1

### PR DESCRIPTION
**Pull request by getdave**
Updates packages for WP 6.7 RC1

Trac ticket: https://core.trac.wordpress.org/ticket/62263

Includes the following Gutenberg PRs:

- Hide inserter pattern/media panel when there are no categories (https://github.com/WordPress/gutenberg/pull/66246)
- Fix empty Patterns inserter when selecting section root when editing Pages in Site Editor (https://github.com/WordPress/gutenberg/pull/66214)
- Resets the Zoom on viewed/edited entity change (https://github.com/WordPress/gutenberg/pull/66232)
- Remove relative position from sidebar tabpanel (https://github.com/WordPress/gutenberg/pull/66267)
- Zoom Out: Disable zoom out toggle button when Style Book is open (https://github.com/WordPress/gutenberg/pull/66228) (https://github.com/WordPress/gutenberg/pull/66233)
- Fix extra scrollbar appearing when searching in the inserter (https://github.com/WordPress/gutenberg/pull/66229)
- Zoom Out: Remove zoom-out toolbar (https://github.com/WordPress/gutenberg/pull/66039) (https://github.com/WordPress/gutenberg/pull/66200)
- Fix scaling animation for device previews (https://github.com/WordPress/gutenberg/pull/66132) (https://github.com/WordPress/gutenberg/pull/66244)
- Block bindings: allow only string field types in bindings. (https://github.com/WordPress/gutenberg/pull/66174)
- Interactivity API: Fix reactivity of undefined objects and arrays added with `deepMerge` (https://github.com/WordPress/gutenberg/pull/66183) (https://github.com/WordPress/gutenberg/pull/66236)
- Block Bindings: Allow label override when it is defined in client registration (https://github.com/WordPress/gutenberg/pull/66160) (https://github.com/WordPress/gutenberg/pull/66237)
- Fix zoom out not persisting while switching between editor and code editor (https://github.com/WordPress/gutenberg/pull/65932)
- Fix/html scale code quality (https://github.com/WordPress/gutenberg/pull/66181)
- Position scaled html within available container space (https://github.com/WordPress/gutenberg/pull/66034)
- Zoom out: fix scaling issues (https://github.com/WordPress/gutenberg/pull/65998)
- BlockCanvas: Fix the height prop and width of the block editor (https://github.com/WordPress/gutenberg/pull/65977)
- Revert "Zoom out: fix scaling issues (https://github.com/WordPress/gutenberg/pull/65998)"
- Memoize pattern objects returned from getAllowedPatterns (https://github.com/WordPress/gutenberg/pull/66162)
- Manual cherry pick for #65627 (https://github.com/WordPress/gutenberg/pull/66119)
- Block Bindings: Bootstrap server sources earlier (https://github.com/WordPress/gutenberg/pull/66058) (https://github.com/WordPress/gutenberg/pull/66167)
- Update site icon and title position (https://github.com/WordPress/gutenberg/pull/66171)
- Performance tests : -  Fix/compare performance with correct branch (https://github.com/WordPress/gutenberg/pull/66196)
- ToggleGroupControl: Don't set value on focus after a reset (https://github.com/WordPress/gutenberg/pull/66151)
- Navigation block: fix block appender size (https://github.com/WordPress/gutenberg/pull/66153)
- Block Bindings: Use `getEntityConfig` instead of `getPostTypes` to get available slugs (https://github.com/WordPress/gutenberg/pull/66101)
- Fix the navigation issue inside cover blocks (https://github.com/WordPress/gutenberg/pull/66093)
- Fix $font-line-height-x-large error (https://github.com/WordPress/gutenberg/pull/66184)
- Fix focus loss when deleting selected block in zoom out mode (https://github.com/WordPress/gutenberg/pull/65761)
- Fix e2e test for padding appender (https://github.com/WordPress/gutenberg/pull/66080)
- Update/replace edit button with enter on selection (https://github.com/WordPress/gutenberg/pull/65760)
- Fix padding appender hook (https://github.com/WordPress/gutenberg/pull/66143)
- Fix site editor back button visual regressions (https://github.com/WordPress/gutenberg/pull/66166)
- Fix rich text toolbar corners (https://github.com/WordPress/gutenberg/pull/66163)
- Add argument with post ID to the editor.savePost hook (https://github.com/WordPress/gutenberg/pull/66165)
- Add missing `post.slug` dependency to `useMemo` (https://github.com/WordPress/gutenberg/pull/66125)
- Polish zoom out inserter (https://github.com/WordPress/gutenberg/pull/66110)
- Fix duotone on parallax/repeated featured image cover blocks (https://github.com/WordPress/gutenberg/pull/65929)
- Revert the manage fonts button. (https://github.com/WordPress/gutenberg/pull/66107)
- Revert "Update z-index hierarchy (https://github.com/WordPress/gutenberg/pull/65626)" (https://github.com/WordPress/gutenberg/pull/66074)
- Data Views list layout: revise for improved text truncation (https://github.com/WordPress/gutenberg/pull/65376)
- Restores setting zoom out mode to useZoomOut hook (https://github.com/WordPress/gutenberg/pull/65999)

---
**This Pull Request is for code review only. Please keep all other discussion in the Trac ticket. Do not merge this Pull Request. See [GitHub Pull Requests for Code Review](https://make.wordpress.org/core/handbook/contribute/git/github-pull-requests-for-code-review/) in the Core Handbook for more details.**

### CODE REVIEW

**Review by kevin940726 — APPROVED** [review]
LGTM 👍

**Comment by getdave** [comment]
cc @ndiego @colorful-tones

**Comment by rejaulalomkhan** [comment]
@getdave Thanks for the excellent PR ✌

**Comment by getdave** [comment]
I tested a few PRs and this seems to match expectations.

File: package.json [hunk 1]
```diff
@@ -32,7 +32,7 @@
 		"@wordpress/e2e-test-utils": "11.8.2",
 		"@wordpress/e2e-test-utils-playwright": "1.8.1",
 		"@wordpress/prettier-config": "4.8.1",
-		"@wordpress/scripts": "30.0.5",
```
Comment (CypherReymax96) [inline package.json:35]: #7600

File: package.json [hunk 2]
```diff
@@ -83,58 +83,58 @@
 		"@wordpress/api-fetch": "7.8.2",
 		"@wordpress/autop": "4.8.1",
 		"@wordpress/blob": "4.8.1",
-		"@wordpress/block-directory": "5.8.8",
```
Comment (CypherReymax96) [inline package.json:86]: #7600 #7598

**Review by CypherReymax96 — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (CypherReymax96) [reply package.json:86]: > #7600 #7598

**Comment by getdave** [comment]
Committed in https://core.trac.wordpress.org/changeset/59263

**Pull request closed without merging**